app/cutemarkdownhighlighter.cpp

```diff
@@ -64,7 +64,7 @@ void CuteMarkdownHighlighter::setYamlHeaderSupportEnabled(bool enabled)
 
 void CuteMarkdownHighlighter::highlightBlock(const QString &textBlock)
 {
-    if (textBlock.isEmpty() && !MarkdownHighlighter::isCodeBlock(previousBlockState())) {
+    if (false && textBlock.isEmpty() && !MarkdownHighlighter::isCodeBlock(previousBlockState())) {
```

Comment:
This is correct, but just remove the line instead of `if (false..`. Empty line is useful in some other contexts as well so yeah

Reply:
Done